src/services/ethereum/utils.ts

```diff
@@ -41,7 +43,18 @@ export async function fetchNetworkGasPrice(chainId: number): Promise<ethers.BigN
 }
 
 async function fetchTransactionMetadata(transaction: IScheduled): Promise<ITransactionMetadata> {
-  const provider = ethers.getDefaultProvider(ethers.providers.getNetwork(transaction.chainId));
+  let provider: ethers.providers.BaseProvider;
+  switch (transaction.chainId) {
+    case ChainId.Arbitrum:
+      provider = new ethers.providers.JsonRpcProvider(ARBITRUM_URI);
+      break;
+    case ChainId.Arbitrum_Rinkeby:
+      provider = new ethers.providers.JsonRpcProvider(ARBITRUM_RINKEBY_URI);
+      break;
+    default:
+      provider = new ethers.providers.JsonRpcProvider(ETHERUM_URI);
```

Comment:
IF we really want to use this in the default then the problem is now that `ETHEREUM_URI` points to only one chain - mainnet, but it must support at least ropsten too. So now you need a new URI for ropsten and special `case`s for mainnet and ropsten